addon/pods/components/rui-avatar/component.js

```diff
@@ -1,50 +1,42 @@
-import Ember from 'ember';
-import layout from './template';
+import Ember from 'ember'
+import layout from './template'
 
-export default Ember.Component.extend({
+const {
+  Component,
+  computed,
+  Logger
+} = Ember
+
+export default Component.extend({
   layout: layout,
-  tagName: 'img',
-  attributeBindings: ['computedSrc:src'],
+  tagName: 'span',
   classNames: ['rui-avatar'],
   classNameBindings: ['sizeComputed'],
 
-  // Defaults
-  computedSrc: Ember.computed('src', function() {
-    // `src` is supplied
-    if (this.get('src') !== '') {
-      return this.get('src');
-    }
-    // `src` is empty
-    if (this.get('src') === '') {
-      Ember.Logger.warn('rui-avatar: You did not specify a `src`, so we have set it to a placeholder image. Please specify a `src`.');
-      return 'http://placehold.it/32?text=%20';
-    }
-  }),
-  src: Ember.computed('src', function() {
-    // `src` is not supplied
-    Ember.Logger.warn('rui-avatar: You did not specify a `src`, so we have set it to a placeholder image. Please specify a `src`.');
-    return 'http://placehold.it/32?text=%20';
+  src: '', //passed in
+  showPlaceholder: computed('src', function() {
+    const src = this.get('src')
+    return src === undefined || src === ''
```

Comment:
hmmm, it could but since null would have to be assigned it doesn't seem to me to be a super relevant case.  It returns from the db as an empty string if there no avatar has been added and undefined if it is not bound via the component.